### ISSUE TITLE
[SUGGESTION] More demucs (models), more fun !

### ISSUE THREAD

**Issue by MarcoRavich** [intel/openvino-plugins-ai-audacity#469]
I've recently tested @CarlGao4's [Demucs GUI](https://github.com/CarlGao4/Demucs-Gui#readme) that allows to inference - even on CPU (but without OpenVINO afaik) many interesting separation models/versions.

Models details from @adefossez's [Demucs-fork-git](https://github.com/adefossez/demucs#readme):

> -  `htdemucs`: first version of Hybrid Transformer Demucs. Trained on MusDB + 800 songs. Default model.
> - `htdemucs_ft`: fine-tuned version of `htdemucs`, separation will take 4 times more time but might be a bit better. Same training set as `htdemucs`.
> - `htdemucs_6s`: 6 sources version of `htdemucs`, with `piano` and `guitar` being added as sources. Note that the `piano` source is not working great at the moment.
> - `hdemucs_mmi`: Hybrid Demucs v3, retrained on MusDB + 800 songs.
> - `mdx`: trained only on MusDB HQ, winning model on track A at the [MDX][mdx] challenge.
> - `mdx_extra`: trained with extra training data (**including MusDB test set**), ranked 2nd on the track B of the [MDX][mdx] challenge.
> - `mdx_q`, `mdx_extra_q`: quantized version of the previous models. Smaller download and storage but quality can be slightly worse.
> - `SIG`: where `SIG` is a single model from the [model zoo](docs/training.md#model-zoo).
> 

I know that the upcoming version will allow their use, anyway I believe that some conversions is needed to.

**Comment by RyanMetcalfeInt8** [comment]
Thanks for the share -- next version will include `htdemucs` (same one we've always had), `htdemucs_ft`, and `htdemucs_6s` (mainly for completeness ... as all sources point out, it's not very good).

It also includes `mdx23c` variant of drum separation.

I haven't yet tried `htdemucs_mmi`, and other variants of `mdx*` -- how do they stack up against htdemucs?

**Comment by MarcoRavich** [comment]
> I haven't yet tried `htdemucs_mmi`, and other variants of `mdx*` -- how do they stack up against htdemucs?
>
Well, there are detailed per-model Quality leaderboards @ MVSEP:
https://mvsep.com/en/quality_checker

(note: it would be really helpful to have metrics - example: https://mvsep.com/quality_checker/entry/8196 - inside the model info tab to understand which to use)


Anyway it would be cool to have all models - or a simple method to use [publicly released weights](https://github.com/ZFTurbo/Music-Source-Separation-Training/releases) - from @ZFTurbo's [Music Source Separation Universal Training Code repo](https://github.com/ZFTurbo/Music-Source-Separation-Training#readme), IMHO.
